lib/bundler/audit/cli.rb

```diff
@@ -34,14 +34,15 @@ class CLI < ::Thor
       method_option :verbose, :type => :boolean, :aliases => '-v'
       method_option :ignore, :type => :array, :aliases => '-i'
       method_option :update, :type => :boolean, :aliases => '-u'
+      method_option :filter, :type => :array, :aliases => '-f'
 
       def check
         update if options[:update]
 
         scanner    = Scanner.new
         vulnerable = false
 
-        scanner.scan(:ignore => options.ignore) do |result|
+        scanner.scan(ignore: options[:ignore], filter: options[:filter]) do |result|
```

Comment:
Noticed when multiple array options are passed the second one defined (the new one filter) returned as an Enumerable when calling with dot syntax `options.filter #=> Enumerable`. Instead using symbol key syntax it returns as expected